src/main/java/com/hcl/appscan/jenkins/plugin/scanners/SoftwareCompositionAnalysis.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * @ Copyright HCL Technologies Ltd. 2023.
+ * LICENSE: Apache License, Version 2.0 https://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package com.hcl.appscan.jenkins.plugin.scanners;
+
+import com.hcl.appscan.jenkins.plugin.auth.JenkinsAuthenticationProvider;
+import hudson.Extension;
+import hudson.RelativePath;
+import hudson.model.ItemGroup;
+import hudson.util.FormValidation;
+import hudson.util.VariableResolver;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.AncestorInPath;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SoftwareCompositionAnalysis extends Scanner {
+    private static final String SOFTWARE_COMPOSITION_ANALYSIS = "Sca"; //$NON-NLS-1$
```

Comment:
This constant is already defined in ScannerConstants, so we should use the existing constant.

Reply:
Okay, sure.